WordPress/gutenberg
# Move block editor settings filter into 6.3 compat folder

## What?


Now that post content attributes have been [added to the block editor settings in core (WP 6.3)](https://core.trac.wordpress.org/ticket/58534), the corresponding Gutenberg filter can be moved to the 6.3 compat folder.

## Testing Instructions





1. With a block theme enabled, edit the Post Content block to give it a custom layout (justify left or right, for example).
2. Open the Post Editor, which should immediately display the correct Post Content layout.

### Testing Instructions for Keyboard


## Screenshots or screencast
